### PR TITLE
SAK-50119 Global cleanup of Deprecated annotation

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

File: citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java [hunk 1]
```diff
@@ -1336,28 +1336,28 @@ protected Map<String,Object> ensureCitationListExists(ParameterParser params,
...
-						log.warn("ServerOverloadException in ensureCitationListExists() {}", e);
+						log.warn("ServerOverloadException in ensureCitationListExists()", e);
 					} catch (PermissionException e) {
 						message = e.getMessage();
-						log.warn("PermissionException in ensureCitationListExists() {}", e);
+						log.warn("PermissionException in ensureCitationListExists()", e);
 					} catch (IdUnusedException e) {
 						message = e.getMessage();
-						log.warn("IdUnusedException in ensureCitationListExists() {}", e);
+						log.warn("IdUnusedException in ensureCitationListExists()", e);
```
Comment (ern) [inline citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java:1360]: Simply catching Exception and then using the exceptions toString will include the exception class name in the message, yielding the same result in the log message.

Comment (ern) [reply citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java:1360]: I would also advocate for a better message, as the current message(s) are unhelpful and expected.

**Comment by ern** [comment]
Instead of all the casting to (Object) to statisfy the vararg it might be better to switch the method signature to
`public String getFormattedMessage(String key, Object[] args)`
 I checked and only saw 2 places that would need updating 1 in SiteAction and 1 in SiteParticpantHelper

File: citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java (hunk 1 above)
Comment (ottenhoff) [reply citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java:1360]: idk this isn't the scope of the change. i'm trying to be helpful and cleanup stuff without refactoring logging in a class that is never used ;)

**Comment by ottenhoff** [comment]
> I checked and only saw 2 places that would need updating 1 in SiteAction and 1 in SiteParticpantHelper


Oh, I think it's probably about 100 places not 2 ;)

**Comment by ern** [comment]
I checked it and it is only 2 and everything built, remember varargs and Object[] have the same signature, so your not updating all of them.

**Comment by ern** [comment]
I fixed up this PR and rebased it

**Review by ern — APPROVED** [review]

**Pull request merged**